dymos/transcriptions/pseudospectral/pseudospectral_base.py

```diff
@@ -501,6 +501,8 @@ def setup_timeseries_outputs(self, phase):
                                                                  output_subset=options['subset'])
```

Comment:
Need to pass `time_units = phase.time_options['units']` here.  Change in time per segment length is just expressed in the same time-units as the phase.